core-web/libs/sdk/client/src/lib/client/sdk-js-client.ts

```diff
@@ -171,6 +175,8 @@ export class DotCmsClient {
                 ...this.config.requestOptions?.headers
             }
         };
+
+        this.content = new Content(this.requestOptions, this.config.dotcmsUrl);
```

Comment:
This needs to be like page and nav, but more important have doc like those.

Reply:
It has docs, it's just encapsulated inside the class